Gemfile.lock

```diff
@@ -305,8 +305,13 @@ GEM
     rubocop-rspec (1.42.0)
       rubocop (>= 0.87.0)
     ruby-progressbar (1.10.1)
-    sentry-raven (3.1.1)
+    sentry-ruby (4.2.2)
```

Comment:
We don't need any of the following?
gem "sentry-rails"
gem "sentry-sidekiq"
gem "sentry-delayed_job"

Reply:
My understanding is that these gems (that depend on sentry-ruby) bring a few little extras, but I don't think we need them. Could be interesting to look into how they could be useful to us.